ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/BlockchainReferenceTestCaseSpec.java

```diff
@@ -166,6 +166,8 @@ public ReferenceTestBlockHeader(
         @JsonProperty("depositsRoot") final String depositsRoot,
         @JsonProperty("dataGasUsed") final String dataGasUsed,
         @JsonProperty("excessDataGas") final String excessDataGas,
```

Comment:
Is there a reason to have dataGasUsed and excessDataGas here?

Reply:
Yes, there is no reference test that has both blobGas as a label and does not have 4788 as a requirement.  This dance covers both the current release of reference tests and future releases.  Look a few lines lower to see how it's used.